app/controllers/uploads_controller.rb

```diff
@@ -84,11 +82,16 @@ def upload_params
 
   def call_load
     file = @upload.upload_file.tempfile
-    data = @upload.csv_type.constantize.load(file, skip_lines: @upload.skip_lines.try(:to_i))
+    skip_lines = @upload.skip_lines.try(:to_i)
+    data = klass.load(file, skip_lines: skip_lines)
 
     @upload.update(ok: data.present? && data.ids.present?)
-    raise StandardError, errors_for_alert([@upload]) unless @upload.ok?
+    raise(StandardError, "There was no saved #{klass} data!") unless @upload.ok?
```

Comment:
I don't think vets.gov has an error message style guide, but I tend to err on the side of business-like. I think we should remove the `!`.